xtests/Makefile.am

```diff
@@ -43,7 +44,8 @@ XTESTS = tst-pam_dispatch1 tst-pam_dispatch2 tst-pam_dispatch3 \
 	tst-pam_access1 tst-pam_access2 tst-pam_access3 \
 	tst-pam_access4 tst-pam_limits1 tst-pam_succeed_if1 \
 	tst-pam_group1 tst-pam_authfail tst-pam_authsucceed \
-	tst-pam_pwhistory1 tst-pam_time1 tst-pam_motd
+	tst-pam_pwhistory1 tst-pam_time1 tst-pam_motd \
+        tst-pam_shells
```

Comment:
Note that tst-pam_shells is indented with spaces while the lines above it are undented with tabs.